client/src/components/Comic.module.scss

```
@@ -0,0 +1,51 @@
+.comic {
```

Maybe this is a little hacky, but it's the easiest way to stop the elements moving around as the image loads
```suggestion
.comic {
  display: flex;
  flex-direction: column;
  justify-content: flex-end;
```